modules/calcite/src/test/java/org/apache/ignite/internal/processors/tx/SqlTransactionsIsolationTest.java

```diff
@@ -218,32 +96,29 @@ public enum ModifyApi {
         }
     });
 
-    /** */
-    private final TransactionIsolation txIsolation = READ_COMMITTED;
-
     /** */
     private final Map<Integer, Set<Integer>> partsToKeys = new HashMap<>();
 
     /** @return Test parameters. */
     @Parameterized.Parameters(
-        name = "modify={0},qryExecutor={1},partitionAwareness={2},mode={3},gridCnt={4},backups={5},commit={6},multi={7},txConcurrency={8}")
+        name = "gridCnt={0},backups={1},partitionAwareness={2},mode={3},execType={4},modify={5},commit={6},multi={7},txConcurrency={8}")
     public static Collection<?> parameters() {
         List<Object[]> params = new ArrayList<>();
 
-        for (CacheMode cacheMode : CacheMode.values()) {
-            for (int gridCnt : new int[]{1, 3}) {
-                int[] backups = gridCnt > 1
-                    ? new int[]{1, gridCnt - 1}
-                    : new int[]{0};
-                for (int backup: backups) {
+        for (int gridCnt : new int[]{1, 3}) {
+            int[] backups = gridCnt > 1
+                ? new int[]{1, gridCnt - 1}
+                : new int[]{0};
+            for (int backup : backups) {
+                for (CacheMode mode : CacheMode.values()) {
                     for (ModifyApi modify : ModifyApi.values()) {
                         for (boolean commit : new boolean[]{false, true}) {
-                            for (boolean mutli : new boolean[] {false, true}) {
+                            for (boolean mutli : new boolean[]{false, true}) {
                                 for (TransactionConcurrency txConcurrency : TransactionConcurrency.values()) {
                                     ExecutorType[] nodeExecTypes = {ExecutorType.SERVER, ExecutorType.CLIENT};
                                     ExecutorType[] thinExecTypes;
 
-                                    if (modify == SQL) {
```

Comment:
Do we need this changes in this patch?

Reply:
The main reason to include test changes in patch is to reduce main patch.
Exclude obvious changes from it.

So I thins yes, we need this :)